frontend/src/App.tsx

```diff
@@ -7,21 +7,33 @@ import { AcademicYearScreen } from "./Onboarding/AcademicYearScreen";
 import { GraduationYearScreen } from "./Onboarding/GraduationYearScreen";
 import { MajorScreen } from "./Onboarding/MajorScreen";
 import { MinorsScreen } from "./Onboarding/MinorsScreen";
+import { Provider } from "react-redux";
+import { Store } from "redux";
+import { PersistGate } from "redux-persist/integration/react";
+import { Persistor } from "redux-persist";
 
-export default class App extends React.Component {
-  render() {
-    return (
-      <Router>
-        <Switch>
-          <Route path="/home" component={HomeWrapper} />
-          <Route path="/name" component={NameScreen} />
-          <Route path="/academicYear" component={AcademicYearScreen} />
-          <Route path="/graduationYear" component={GraduationYearScreen} />
-          <Route path="/major" component={MajorScreen} />
-          <Route path="/minors" component={MinorsScreen} />
-          <Route path="/" component={Onboarding} />
-        </Switch>
-      </Router>
-    );
-  }
-}
+export const App = ({
+  store,
+  persistor,
```

Comment:
Should there be type annotations here?

Reply:
There are. On lines 19-20 is the type annotation. Kind of a pain but this is the syntax that works with typescript. First define the object without types, then define the object with types.

Reply:
OH I see, confused that with the return type. Thanks!